docs/cli/usage/permissions-boundary.md

```diff
@@ -0,0 +1,51 @@
+---
+title: IAM Permissions Boundary for Amplify-generated roles
+description: Apply a Permissions Boundary to all IAM Roles created by Amplify CLI.
+---
+
+To set the maximum permissions that can be granted to IAM Roles created by Amplify CLI, configure a [Permissions Boundary](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html) for the project. Then, Amplify-generated IAM roles can perform only the actions that are allowed by both the roles’ policies and Permissions Boundary. You can configure a different Permissions Boundary for each environment. For example, this enables you to deny a dev environment all access to a prod environment's resources.
+
+The IAM Permissions Boundary will apply to all IAM Roles created by Amplify. This includes the "auth role" assumed by users that log into the app and the "unauth role" assumed by guest users. It also applies to Lambda execution roles, Cognito user group roles, and any role configured in a [custom resource stack](~/cli/usage/customcf.md).
+
+Amplify CLI does set up a new IAM Policy to use as a Permissions Boundary. A Permissions Boundary is an IAM Policy and can be created following the guide [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create-console.html). This IAM Policy, to be used as a Permissions Boundary, must be configured outside of Amplify CLI. This is usually part of an AWS Organization rule or other corporate governance requirement. Once you have created an IAM Policy to use as a Permissions Boundary, copy the IAM Policy ARN for the next steps.
```

Comment:
First sentence should say `Amplify CLI does not set up...`. Although I think the first sentence can be removed as the 3rd sentence basically says the same thing